### PR TITLE
feat: display more data on the node overview page

### PR DESCRIPTION
**Pull request by Unix4ever**
Now the page has an additional panel with:

- machine stage from the `ClusterMachineStatus`.
- last active time from `MachineStatusLink`.
- unmet conditions from Talos `MachineStatus`.
- CPU and memory information from the Omni `MachineStatus`.

Additionally display labels and allow editing them in the separate panel.

Also add missing service phases to the service events UI.

Fixes: https://github.com/siderolabs/omni/issues/219

### CODE REVIEW

**Comment by Unix4ever** [comment]
image
image

File: internal/backend/runtime/cosi/cosi.go [hunk 1]
```diff
@@ -169,7 +169,9 @@ func WatchLegacy(ctx context.Context, st state.State, md resource.Metadata, out
 					return fmt.Errorf("failed to encode resource to send back to the client %w", err)
 				}
 
-				items = append(items[:index], items[index+1:]...)
+				if items != nil {
+					items = append(items[:index], items[index+1:]...)
```
Comment (DmitriyMV) [inline internal/backend/runtime/cosi/cosi.go:173]:
```suggestion
					items = slices.Delete(items, index, index+1)
```

**Review by DmitriyMV — APPROVED** [review]
Small nit.

**Review by utkuozdemir — APPROVED** [review]

**Comment by utkuozdemir** [comment]
Just an idea, you can consider showing the secure boot status on the overview page as well. I put it on the machines screen, but here also could be nice, with some color maybe.

**Comment by Unix4ever** [comment]
> Just an idea, you can consider showing the secure boot status on the overview page as well. I put it on the machines screen, but here also could be nice, with some color maybe.

Sure, not a problem.
I'm going to drop `Hardware` title though to make it occupy the same size

**Comment by Unix4ever** [comment]
image

**Comment by utkuozdemir** [comment]
> image

beautiful

**Pull request merged**